libs/designsystem/item/src/item.component.ts

```diff
@@ -41,4 +65,21 @@ export class ItemComponent {
       event.preventDefault();
     }
   }
+
+  ngAfterContentInit(): void {
+    if (this.label) {
+      if (this.checkbox) {
+        this.moveLabel(this.checkbox.nativeElement.querySelector('ion-checkbox'));
+      }
+      if (this.radio) {
+        this.moveLabel(this.radio.nativeElement.querySelector('ion-radio'));
+      }
+    }
+  }
+
+  private moveLabel(newParent: HTMLElement) {
+    const labelElement = this.label.nativeElement;
+    this.renderer.removeChild(labelElement.parentElement, labelElement);
+    this.renderer.appendChild(newParent, labelElement);
+  }
```

Comment:
Can we extend this method – or use a separate method – so label element is never placed in the default slot? Ie., the label element is always placed in either `slot="start"` or `slot="end"`. That would (probably) solve some styling issues and make everything a bit more consistent.

Not sure about it, but it's a thought 😊

Reply:
Disussed with team, we'll leave it as is for now 👍🏻